config/settings/rdb.q

```diff
@@ -31,9 +31,9 @@ subfiltered:0b      //allows subscription filters to be loaded and applied in th
 connectonstart:1b   //rdb connects to tickerplant as soon as it is started
 
 \d .ds
-datastripe:1b
+datastripe:0b
 period:0D01                                    
-periodstokeep:4
+/periodstokeep:4
```

Comment:
Are these supposed to be commented out (same for next file

Reply:
Yes, this has been moved to the default.q file. I will delete these lines